StripeFinancialConnections/StripeFinancialConnections/Source/Common/HostController.swift

```diff
@@ -39,7 +39,8 @@ class HostController {
 
     // MARK: - Properties
 
-    private let api: FinancialConnectionsAPIClient
+    private let apiClient: FinancialConnectionsAPIClient
+    private let analyticsApiClient: STPAPIClient
```

Comment:
oh that is quite unfortunate that we have to do that haha...I didn't realize.

even the idea that one can't use the analytics client "out of the box" where you have to pass in an API client in each call is very odd too

🤷

Reply:
I wonder if you can just change `private let apiClient: FinancialConnectionsAPIClient` to `private let apiClient: STPAPIClient ` and it works just as well? without the 2 variables?

Reply:
I guess the key insight is that you don't really need to use `FinancialConnectionsAPIClient` (just use `STPAPIClient`) unless there are some unit tests that break where stubbing no longer works

Reply:
I think I can just keep only the `STPAPIClient` here! It is a bummer that the relationship is `STPAPIClient` conforms to `FinancialConnectionsAPIClient`, and not the other way around 🤷‍♂️